docs/.vuepress/theme/components/Navigation.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="top-wrapper">
+    <router-link to="/">
+      <img
+        class="header-logo"
+        src="../images/ethereum-logo-wireframe.png"
+        alt="Ethereum Logo"
+      />
+    </router-link>
+    <NavLinks
+      :isDarkMode="isDarkMode"
+      :isMobileNavVisible="isMobileNavVisible"
+      :handleNavToggle="handleNavToggle"
+      @nav-toggle="handleNavToggle"
+      @dark-mode-toggle="$emit('dark-mode-toggle')"
+    />
+
+    <span class="icon-link icon-menu" tabindex="0">
+      <icon
+        name="menu"
+        class="icon-menu"
+        @click.native="handleNavToggle(true)"
+      />
+    </span>
+  </div>
+</template>
+
+<script>
+import NavLinks from './NavLinks.vue'
+import SearchBox from './SearchBox.vue'
```

Comment:
Don't think this import is needed - you don't use the `SearchBox` component in this file?